setup.py

```diff
@@ -68,12 +70,101 @@ def read_file(a_file):
     return content
 
 
+class PytestCommand(setuptools.Command):
+    """
+    Base class for setup.py commands for executing tests for this package
+    using pytest.
+
+    Note on the class name: Because distutils.dist._show_help() shows the class
+    name for the setup.py command name instead of invoking get_command_name(),
+    the classes that get registered as commands need to have the command name.
```

Comment:
Getting picky.  Change "need to have" to "must"

Reply:
DONE